cfgov/cfgov/settings/base.py

```diff
@@ -64,18 +74,14 @@
         }
     },
     {
-        'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'NAME': 'wagtail-env',
+        'BACKEND': 'django.template.backends.jinja2.Jinja2',
+        'DIRS': [Path(os.path.dirname(__file__)).ancestor(2) + '/v1/jinja2/v1'],
         'APP_DIRS': True,
```

Comment:
by setting the APP_DIR to true, it should find the templates residing in the jinja2 directory.. were you seeing something different?